astroplan/utils.py

```diff
@@ -148,6 +148,9 @@ def time_grid_from_range(time_range, time_resolution=0.5*u.hour):
     times : `~astropy.time.Time`
         Linearly-spaced sequence of times
     """
+    if len(time_range != 2):
```

Comment:
Damn, I should stop doing anything and just go home, it appears that I make more typos today than working lines.